blue-note/functions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Register block styles.
+ * TODO: Add CSS, repeat for both featured image and image blocks
```

Comment:
I wonder if this should be a style for the query loop instead of the images and have it apply the style for the odd/even images in the loop. Are there any issues in Gutenberg about styling differently each element of the loop? historically we've done that using more than one loop and offsetting them, but that's clunky at best and a power user move. Maybe this is the moment to find a solution for this in the editor.

Reply:
Yes I think we will run into issues. Will be interesting to see.

Reply:
There is one section that was in the middle of a page, that was like a quote with a slanted image: https://github.com/WordPress/community-themes/issues/6

Reply:
I see, do you think we should be have a "rotate" option for certain blocks in the editor?

Reply:
I'm not fully convinced that this should be a block option in core or Gutenberg, it is not so common.

Reply:
https://github.com/WordPress/gutenberg/issues/40903 I think it's similar to this issue

Reply:
I have added the slanted image style variation to featured images, images and the query.
But I am not sure yet how to do the duotone for the query.

I also found that the block style is not working in the site editor, but is working in the block editor and front. I think this issue is already reported, I just need to find it.

Reply:
let's get the general issues done here and we can focus on separate PRs on each template, that way is easier to review and for others to contribute too.

Reply:
I suppose we could hard code the duotone image filter...

Reply:
are you trying to have the block style do both the rotation and the duotone filter? I'm not sure I understand the problem.

And about the query loop, if you mean it for the home page, where every image has a different angle and color, we need three query loops one after the other, then the second loop will be offset by 1 post and the third by two posts.

Reply:
I believe it can be done with CSS only, without multiple queries

Reply:
I would think anything is easier than managing 3 queries (in 3 template since the same feature are in the archive and search).
But if this has been done in .com themes before, and users have not brought it up as an issue, I guess it is a non issue.

The duotone palette can be updated in the Styles section, right? Or is that. Gutenberg only feature?

Reply:
We've done this a a few times, yes. In the end, power users will use the list view and see the three queries and non power users will simply click on a block and expect to be able to change the duotone and be able to do so. It's not ideal but it works.

The duotone palette can't be changed yet, I'm working on a PR for that but I don't know how long it will take, there's some design of the UI that needs to happen, right now it can only be changed by modifying theme.json.